content/docs/components/serving/seldon.md

```diff
@@ -47,6 +47,115 @@ spec:
 
 Save the above resource and apply it with `kubectl`.
 
-## Examples
+## Simple Example
```

Comment:
For consistency within the page and with the Kubeflow style guide, change this heading to sentence case: 

```
## Simple example
```